src/api/forms/draft-form-definition-repository.js

```diff
@@ -16,11 +17,12 @@ const formBucketName = config.get('formDefinitionBucketName')
 /**
  * Gets the path to a form definition file for a given form ID
  * @param {string} formId - the form ID
+ * @param {'draft'|'live'} state - the form state
```

Comment:
Tiny little formatting preference

```suggestion
 * @param {'draft' | 'live'} state - the form state
```

Reply:
Resolved with 143e9cd20fb194525b54826b417f88b54311d695